yarn-project/acir-simulator/src/client/private_execution.test.ts

```diff
@@ -504,7 +509,7 @@ describe('Private Execution test suite', () => {
       // Check the commitment read request was created successfully.
       const readRequests = result.callStackItem.publicInputs.readRequests.filter(field => !field.equals(Fr.ZERO));
       expect(readRequests).toHaveLength(1);
-      expect(readRequests[0]).toEqual(commitment);
```

Comment:
For the renaming PR: still some conflation of 'commitment' and 'note_hash' in places

Reply:
* https://github.com/AztecProtocol/aztec-packages/issues/1408
* https://github.com/AztecProtocol/aztec-packages/issues/1409